### PR TITLE
fixit: clean up run samples

### PR DESCRIPTION
**Pull request by averikitsch**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by engelke — APPROVED** [review]

File: run/filesystem/main.py [hunk 1]
```diff
@@ -22,72 +22,88 @@
...
 
-    html += '</body></html>\n'
+    html += "</body></html>\n"
     return html
 
 
 def write_file(mnt_dir, filename):
-    '''Write files to a directory with date created'''
+    """Write files to a directory with date created.
+    Args:
```
Comment (rsamborski) [inline run/filesystem/main.py:71]: nit: add an empty line before `Args`

File: run/filesystem/main.py [hunk 2]
```diff
@@ -22,72 +22,88 @@
...
-        f.write(f'This test file was created on {date}.')
+    file_date = "{dt:%a}-{dt:%b}-{dt:%d}-{dt:%H}:{dt:%M}-{dt:%Y}".format(dt=date)
+    with open(f"{mnt_dir}/{filename}-{file_date}.txt", "a") as f:
+        f.write(f"This test file was created on {date}.")
 
 
 def read_file(full_path):
-    '''Read files and return contents'''
+    """Read files and return contents.
+    Args:
```
Comment (rsamborski) [inline run/filesystem/main.py:83]: nit: ditto, add an empty line

File: run/filesystem/main.py [hunk 3]
```diff
@@ -22,72 +22,88 @@
...
+        full_path: the path to the file
+    Returns:
```
Comment (rsamborski) [inline run/filesystem/main.py:85]: nit: ditto, add an empty line before `Returns`

File: run/idp-sql/database.py [hunk 4]
```diff
@@ -33,6 +33,10 @@
 
 
 def init_connection_engine() -> sqlalchemy.engine.base.Engine:
+    """Initializes a connection pool for a Cloud SQL instance of PostgreSQL.
+    Returns:
```
Comment (rsamborski) [inline run/idp-sql/database.py:37]: nit: ditto, add an empty line

File: run/idp-sql/database.py [hunk 5]
```diff
@@ -65,6 +69,12 @@ def init_connection_engine() -> sqlalchemy.engine.base.Engine:
 def init_tcp_connection_engine(
     db_config: dict[str, type[NullPool]]
 ) -> sqlalchemy.engine.base.Engine:
+    """Initializes a TCP connection pool for a Cloud SQL instance of PostgreSQL.
+    Args:
```
Comment (rsamborski) [inline run/idp-sql/database.py:73]: nit: ditto, add an empty line

File: run/idp-sql/database.py [hunk 6]
```diff
@@ -65,6 +69,12 @@ def init_connection_engine() -> sqlalchemy.engine.base.Engine:
...
+        db_config: a dictionary with connection pool config
+    Returns:
```
Comment (rsamborski) [inline run/idp-sql/database.py:75]: nit: ditto, add an empty line

File: run/idp-sql/database.py [hunk 7]
```diff
@@ -98,6 +108,12 @@ def init_tcp_connection_engine(
 def init_unix_connection_engine(
     db_config: dict[str, int]
 ) -> sqlalchemy.engine.base.Engine:
+    """Initializes a Unix socket connection pool for a Cloud SQL instance of PostgreSQL.
+    Args:
```
Comment (rsamborski) [inline run/idp-sql/database.py:112]: nit: ditto, add an empty line

File: run/idp-sql/database.py [hunk 8]
```diff
@@ -98,6 +108,12 @@ def init_tcp_connection_engine(
...
+        db_config: a dictionary with connection pool config
+    Returns:
```
Comment (rsamborski) [inline run/idp-sql/database.py:114]: nit: ditto, add an empty line

File: run/idp-sql/database.py [hunk 9]
```diff
@@ -131,32 +147,40 @@ def init_unix_connection_engine(
...
+                "uid VARCHAR(128) NOT NULL, "
+                "PRIMARY KEY (vote_id)"
+                ");"
+            )
+        )
 
 
 def get_index_context() -> dict[str, Any]:
+    """Query PostgreSQL database and transform data for UI.
+    Returns:
```
Comment (rsamborski) [inline run/idp-sql/database.py:172]: nit: ditto, add an empty line

File: run/idp-sql/database.py [hunk 10]
```diff
@@ -180,6 +204,12 @@ def get_index_context() -> dict[str, Any]:
 
 
 def save_vote(team: str, uid: str, time_cast: datetime.datetime) -> None:
+    """Save a vote into the PostgreSQL database.
+    Args:
```
Comment (rsamborski) [inline run/idp-sql/database.py:208]: nit: ditto, add an empty line

File: run/idp-sql/middleware.py [hunk 11]
```diff
@@ -59,6 +65,16 @@ def decorated_function(*args: a, **kwargs: a) -> a:
...
     logger: structlog._loggers.PrintLogger, log_method: str, event_dict: dict
 ) -> dict:
+    """A structlog processor for mapping fields to Cloud Logging.
+    Learn more at https://www.structlog.org/en/stable/processors.html
+
+    Args:
+        logger: A logger object.
+        log_method: The name of the wrapped method.
+        event_dict:Current context together with the current event.
+    Returns:
```
Comment (rsamborski) [inline run/idp-sql/middleware.py:75]: nit: ditto, add an empty line

File: run/idp-sql/middleware.py [hunk 12]
```diff
@@ -68,6 +84,10 @@ def field_name_modifier(
 
 
 def getJSONLogger() -> structlog._config.BoundLoggerLazyProxy:
+    """Initialize a logger configured for JSON structured logs.
+    Returns:
```
Comment (rsamborski) [inline run/idp-sql/middleware.py:88]: nit: ditto, add an empty line

**Review by rsamborski — APPROVED** [review]
Approved with couple nits

**Pull request merged**